doc/getting-started.rst

```diff
@@ -0,0 +1,565 @@
+=============================
+Getting started with Buildout
+=============================
+
+.. contents::
+
+.. note::
+
+   In the Buildout documentation, we'll use the word
+   *buildout* to refer to:
+
+   - The Buildout software
+
+     We'll capitalize the word when we do this.
+
+   - A particular application of Buildout, which has a buildout
+     configuration.
+
+     We'll use lower case to refer to these.
+
+   - A ``buildout`` section in a Buildout configuration (in a
+     particular buildout).
+
+     We'll use a lowercase fix-width font for these.
+
+First steps
+===========
+
+The easiest way to install Buildout is with pip::
+
+  pip install zc.buildout
+
+To use Buildout, you need to provide a Buildout configuration. Here is
+a minimal configuration::
+
+  [buildout]
+  parts =
+
+A minimal (and useless) Buildout configuration has a ``buildout`` section
+with a parts option.  If we run Buildout::
+
+  buildout
+
+Four directories are created:
+
+bin
+  A directory to hold executables.
+
+develop-eggs
+  A directory to hold develop egg links. More about these later.
+
+eggs
+  A directory that hold installed packages in egg [#egg]_ format.
+
+parts
+  A directory that provides a default location for installed parts.
+
+Buildout configuration files use an `INI syntax
+<https://en.wikipedia.org/wiki/INI_file>`_ [#configparser]_.
+Configuration is arranged in sections, beginning with section names in square
+brackets. Section options are names, followed by equal signs, followed
+by values.  Values may be continued over multiple lines as long as the
+continuation lines start with whitespace.
+
+Buildout is all about building things and the things to be built are
+specified using *parts*.  The parts to be built are listed in the
+``parts`` option.  For each part, there must be a section with the same
+name that names the software to build the part and optionally provides
```

Comment:
`name that names` sounds weird...
How about `...that specifies the software to build the part`.

Also, I think `optionally` is not strictly necessary here.